ui/pages/notifications/index.scss

```
@@ -96,13 +96,21 @@
     }
 
     .unread {
+      flex-shrink: 0;
       background-color: var(--color-primary-default);
     }
 
     &__details {
       display: flex;
       flex-direction: column;
       margin-left: 12px;
+      width: 100%;
+
+      &__message {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
```

```suggestion
        overflow-wrap: anywhere
```

We would prefer showing the full text instead of truncating it!